loops.js

```diff
@@ -32,7 +36,13 @@
 
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over. Google daarom eens naar 'mdn webdocs remainder operator'.
 // ==========================================
-
+for (let i = 0; i < 15; i++)    {
+    if( i % 2 == 0)    {
+        console.log( i + " is een even getal");
+    } else  {
+        console.log(i + " is een oneven getal");
```

Comment:
Bij de eerste console.log heb je een spatie tussen ( en i, bij de tweede niet. Het is netter om deze spatie weg te halen.